### DISCUSSION TITLE
script is not throwing any error or executing

### DISCUSSION THREAD

**Discussion by azeemgujjar** [spatie/ssh#46] [Q&A]
I am trying to run SSH command on ec2 linux instance, but it is not running or not even throwing any error
I wrote this command in laravel command and running script through terminal.
Please help
```
$process = Ssh::create('ec2-user', $publicIP)->usePrivateKey(public_path('Web-SDP.pem'))->execute(
                'streamlink twitch.tv/nickmercs best,1080p60,1080,720,480,worst --output ~/test.mp4 --retry-streams 10 --force --retry-max 360 --retry-open 20 --twitch-disable-hosting --twitch-disable-ads'
            );

            echo $process->isSuccessful();
            echo $process->getOutput();
```

**Comment by alexevladgabriel** [comment]
Hey @azeemgujjar, did you found a solution, I have the same problem in my Windows Machine, I don't see any connection made with the dedicated server. If you succeed let me know, please.
